webapp/package.json

```
@@ -102,7 +107,11 @@
       "^.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "identity-obj-proxy",
       "^.+\\.(css|less|scss)$": "identity-obj-proxy",
       "^.*i18n.*\\.(json)$": "<rootDir>/tests/i18n_mock.json",
-      "^bundle-loader\\?lazy\\!(.*)$": "$1"
+      "^bundle-loader\\?lazy\\!(.*)$": "$1",
+      "^@mattermost/types/(.*)$": "<rootDir>/mattermost-webapp/webapp/platform/types/src/$1",
+      "^@mattermost/client$": "<rootDir>/mattermost-webapp/webapp/platform/client/src/index",
+      "^@mattermost/client/(.*)$": "<rootDir>/mattermost-webapp/webapp/platform/client/src/$1",
+      "^mattermost-redux(.*)$": "<rootDir>/mattermost-webapp/webapp/channels/src/packages/mattermost-redux/src$1"
```

Does this also work?

```suggestion
      "^@mattermost/client/(.*)$": "<rootDir>/mattermost-webapp/webapp/platform/client/src/$1",
      "^mattermost-redux/(.*)$": "<rootDir>/mattermost-webapp/webapp/channels/src/packages/mattermost-redux/src/$1"
```